### PR TITLE
Don't do unnecessary null to empty string conversion

### PR DESCRIPTION
**Pull request by sarav**
We can just check for null instead of an empty string.

### CODE REVIEW

**Comment by Blaisorblade** [comment]
BTW, nullness checkers might help, and this question lists a few reasonable checkers: http://stackoverflow.com/q/14076296/53974. 
[This link](http://types.cs.washington.edu/checker-framework/current/checker-framework-manual.html#other-tools) compares a few.

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java [hunk 1]
```diff
@@ -946,10 +946,11 @@ private Pokemon interpretWhichPokemonUserInput() {
      * screenshot, and then deletes the screenshot.
      */
     private void deleteScreenShotIfRequired() {
-        if (batterySaver && !ssFile.isEmpty()) {
+        if (batterySaver && ssFile != null) {
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: So it turns out the existing code can fail on a NullPointerException :-(.

Comment (sarav) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Huh? Not sure I understand. Bug in my code or did I fix something?

Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Turned out `ssFile.isEmpty()` can crash so I went checking this PR.

**Pull request closed without merging**